nav2_collision_monitor/test/collision_monitor_node_test.cpp

```diff
@@ -908,6 +908,7 @@ TEST_F(Tester, testCrossOver)
   // 1. Obstacle is not in the slowdown zone, but less than TIME_BEFORE_COLLISION (ahead in 1.5 m).
   // Robot should approach the obstacle.
   publishPointCloud(2.5, curr_time);
+  publishRange(2.5, curr_time);
```

Comment:
This is very strange - what was happened here: why the test became to fail? Initially, it was developed to pointcloud to be enough to trigger the approach model. What is the reason, why pointcoud was not enough to do that?

Reply:
It is actually a proof that this PR kind of works ,)
New default behavior is to stop if a source is invalid. In this test, a range source was declared but was never published. So as we have a default source timeout of 5s, the test was failing. (I think)

Reply:
Ah, yes this takes place, as previously in this testcase range source was added: `addSource(RANGE_NAME, RANGE)`